--- t/00-compile.t
+++ t/00-compile.t
@@ -3,7 +3,7 @@
 
 # this test was generated with Dist::Zilla::Plugin::Test::Compile 2.030
 
-use Test::More 0.94 tests => 1 + ($ENV{AUTHOR_TESTING} ? 1 : 0);
+use Test::More 0.47 tests => 1 + ($ENV{AUTHOR_TESTING} ? 1 : 0);
 
 
 
@@ -41,4 +41,3 @@
 
 is(scalar(@warnings), 0, 'no warnings found') if $ENV{AUTHOR_TESTING};
 
-BAIL_OUT("Compilation problems") if !Test::More->builder->is_passing;
